src/app/test/db/mongo/models/models_test.py

```diff
@@ -654,138 +652,3 @@ def test_user_delete_cascades_to_user_token(self, user_token):
         assert len(UserToken.objects.filter(id=user_token.id)) == 1
         user_token.user.fetch().delete()
         assert len(UserToken.objects.filter(id=user_token.id)) == 0
-
```

Comment:
These tests were moved into `test/db/mongo/models/garden_model_test.py`. There is no other change in this file.